Packs/FortiSIEM/IncidentFields/incidentfield-Incident_Last_Seen.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@OfekMQmasters Either add a prefix to the name, id and cliname or add this field to the "CommonTypes" pack.

Reply:
Done